### DISCUSSION TITLE
Init PostgreSQL with other Encoding

### DISCUSSION THREAD

**Discussion by 3xc3ption** [ddev/ddev#3913] [General]
Hi together, 
the implemantation of PostgreSQL works fine, but if want to changed the encoding or creating a database with other encoding, then i got an error.

`psql -c 'CREATE DATABASE "tempDB" WITH OWNER = "db" LC_COLLATE = '\''de_DE.utf8'\'' TEMPLATE template0;'`

Error:
invalid locale name: "de_DE.utf8"
or 
7 ERROR: invalid byte sequence for encoding "UTF8": 0xfc

![image](https://user-images.githubusercontent.com/3159761/173581953-81e65a82-864f-451e-9dff-777b94c285b4.png)

How i can change the encoding on init the Postgres DB?

thanks for support

**Comment by rfay** [comment]
Please try adding to your .ddev/config.yaml `dbimage_extra_packages: ["locales-all"]` and see if that helps. From https://stackoverflow.com/a/49804641/215713

Seems that the docker container needs the locale created.
